webpack/assets/javascripts/react_app/Root/ReactApp.js

```diff
@@ -19,7 +20,10 @@ const ReactApp = ({ layout, metadata, toasts }) => {
         <ApolloProvider client={apolloClient}>
           <ConnectedRouter history={history}>
             <Layout data={layout}>
-              <AppSwitcher />
+              <>
+                <ToastsList railsMessages={toasts} />
+                <AppSwitcher />
+              </>
```

Comment:
Why do we need a fragment wrapper here?
```suggestion
<Layout data={layout}>
   <ToastsList toasts={toasts} />
   <AppSwitcher />
</Layout>
            
```

Reply:
thanks, no need :)